content/tutorial/tutorial.md

```diff
@@ -332,6 +332,15 @@ After installing React DevTools, you can right-click on any element on the page,
 3. Click "Change View" and then choose "Debug mode".
 4. In the new tab that opens, the devtools should now have a React tab.
 
+### A Quick Summary Before Moving On
```

Comment:
I always like a quick summary in between sections. For one thing, it gives me a pause to reflect on what I've learned. Also, it addresses the core things that I need to be aware of.